scripts/backup-sr-metadata.py

```diff
@@ -7,21 +7,15 @@
 import sys
 import getopt
```

Comment:
This file is not packaged by any of xapi packages, nor get installed into dom0,
Is it obsoleted?  Maybe better to remove the file.